frontend/packages/console-dynamic-plugin-sdk/package.json

```diff
@@ -16,6 +16,7 @@
     "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'"
   },
   "devDependencies": {
+    "@openshift/dynamic-plugin-sdk": "1.0.0-alpha14",
```

Comment:
We'll need to bump this version once https://github.com/openshift/dynamic-plugin-sdk/pull/146 gets merged.

Reply:
PR updated, this should be fixed.